src/pages/api/listings/scout/[id].ts

```diff
@@ -110,17 +109,17 @@ async function scoutTalent(req: NextApiRequestWithUser, res: NextApiResponse) {
       },
     });
 
-    if (prevScouts[0]) {
```

Comment:
what was this condition for?

Reply:
this was to be reverted after prathiks test, will do it

Reply:
reverted